contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.3.0--3.4.0.sql

```diff
@@ -3126,4 +3126,39 @@ CALL sys.analyze_babelfish_catalogs();
 -- Reset search_path to not affect any subsequent scripts
 SELECT set_config('search_path', trim(leading 'sys, ' from current_setting('search_path')), false);
 
+-- Change the owner of the current database.
+-- This is a wrapper around ALTER AUTHORIZATION ON DATABASE::
+CREATE OR REPLACE PROCEDURE sys.sp_changedbowner(
+	IN "@loginame" sys.sysname,
+	IN "@map"      sys.VARCHAR(5) DEFAULT NULL) -- this parameter is ignored in T-SQL
+LANGUAGE 'pltsql'
+AS $$
+BEGIN
+	DECLARE @cmd sys.NVARCHAR(300)
+	DECLARE @db  sys.sysname = DB_NAME()
+
+	-- For a NULL login name, do nothing
+	IF @loginame IS NULL
+	BEGIN
+		RETURN
+	END
+
+	IF (@db = 'master') OR (@db = 'tempdb')
```

Comment:
I am wondering why not add `model` and `distribution` database names in the check

Reply:
Babelfish has not implemented model and distribution. Will adjust the error message to remove those.